Source/SwiftLintBuiltInRules/Rules/Idiomatic/FinalClassRule.swift

```diff
@@ -0,0 +1,64 @@
+import SwiftSyntax
+
+@SwiftSyntaxRule
+struct FinalClassRule: Rule {
```

Comment:
Perhaps extends the name to `StaticOverFinalClass` and the identifier to `static_over_final_class` accordingly.